homework/tests/IrenosHomework.spec.js

```diff
@@ -0,0 +1,168 @@
+const { test, expect } = require('@playwright/test');
+
+const data = [
+  'Prototype',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9'
+]
+
+
+data.forEach(version => {
+  test.describe(version + ': Add', () => {
+    test('Adding 5 and 7 result is 12', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
```

Comment:
Since this is a common step throughout all test cases - you could use `beforeEach` hook to navigate to this URL